person.rb

```diff
@@ -1,16 +1,22 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rental'
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
 
```

Comment:
- [ ] As requested by the previous reviewer, you are using rental as your instance variable, not rentals so kindly fix one of the attributes for consistency and to avoid errors. :blush: